scripts/test-utils/harmlessUploadDemos.js

```diff
@@ -0,0 +1,53 @@
+const { ApiPromise, WsProvider, Keyring } = require('@polkadot/api');
+const assert = require('assert/strict');
+const { exec } = require('child_process');
+const { checkInit, uploadProgram } = require('./util.js');
+
+async function main(demoPaths) {
+    // Create connection
+    const provider = new WsProvider('ws://127.0.0.1:9944');
+    const api = await ApiPromise.create({ provider });
+
+    // Create alice account
+    const account = new Keyring({ type: 'sr25519' }).addFromUri('//Alice');
+    // Check that it is root
+    assert.ok((await api.query.sudo.key()).eq(account.addressRaw));
+
+    const isInitialized = checkInit(api);
+
+    for (const pathToDemo of demoPaths) {
+        console.log(`Uploading demo: ${pathToDemo}`);
+
+        const [_programId, messageId] = await uploadProgram(api, account, pathToDemo);
+        await isInitialized(messageId);
+    }
+  };
+
+  const args = process.argv.slice(2);
+  assert.notEqual(args.length, 0, "No demo paths passed");
+  let exitCode = undefined;
+
+  main(args)
+    .then(() => {
+      exitCode = 0;
+    })
+    .catch((error) => {
+      console.error(`JS_TEST: ${error}`);
+      exitCode = 1;
+    })
+    .finally(() => {
+      exec('kill -9 $(pgrep -a gear-node)', (err, stdout, stderr) => {
```

Comment:
Do we really need to kill node after each js test execution ? May be better make one kill in CI after all tests are passed

Reply:
In that case we should take care of node state: for example, that's not fully correct to test anything else after runtime upgrade test
Also, than we should take care of killing node, while running js tests locally via `gear.sh`

There is no overhead, cause we use pre-compiled node binary

Reply:
We can make one function in test.sh which will run node, then run all js scripts, then kill it. Node state is not so bad I think, because it adds some variability

Reply:
What do you think about adding flag --kill-node to scripts?
like npm run upload --kill-node ..other args

and we can leave this tests as it is atm, but kill node in CI just once?